src/python/grpcio_csds/grpc_csds/__init__.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Channelz debug service implementation in gRPC Python."""
+
+from grpc._cython import cygrpc
+
+from google.protobuf import json_format
+try:
+    from envoy.service.status.v3 import csds_pb2, csds_pb2_grpc
+except ImportError:
+    from src.proto.grpc.testing.xds.v3 import csds_pb2, csds_pb2_grpc
```

Comment:
I'm surprised that the only suffixes these import paths have in common is `v3`. If there were more there, I would recommend that you use a combination of `imports` and [`strip_prefixes`](https://github.com/grpc/grpc/blob/68aed165a746450c907a35a07aa342e49d1032e3/bazel/python_rules.bzl#L221).

This isn't ideal, but it's probably okay. The only problem I can foresee is issues not being caught because of behavior differences between the Bazel version and the setuptools version.

Reply:
`imports` and `strip_prefixes` can shorten the proto import path, but the proto import path itself is not the same for our copied version and ground truth... I will keep this as the least worst solution.

I also tried to use `@envoy_api` to let Bazel build from ground truth. However, Envoy's build system doesn't provide generated Python gRPC code (the `*_pb2_grpc` ones).